README.md

```diff
@@ -5,140 +5,14 @@ hardware targets using the Yocto project.
-    export manifest="pelux-intel.xml"
-    ```
-
-4. Do repo init
-
-    ```bash
-    vagrant ssh -c "/vagrant/ci-scripts/do_repo_init ${manifest}"
-    ```
-
-5. Setup bitbake with correct local.conf and bblayers.conf
-
-    ```bash
-    vagrant ssh -c "TEMPLATECONF=${yoctoDir}/sources/meta-pelux-bsp-intel/conf \
-        /vagrant/vagrant-cookbook/yocto/fetch-sources-for-recipes.sh \
-        ${yoctoDir} \
-        ${bitbake_image}"
-    ```
-
-6. Bitbake the PELUX image
-
-    ```bash
-    vagrant ssh -c "/vagrant/vagrant-cookbook/yocto/build-images.sh \
-        ${yoctoDir} \
-        ${bitbake_image}"
-    ```
-
-7. Move the built images to the host
-
-    ```bash
-    vagrant scp :${yoctoDir}/build/tmp/deploy/images ../images
-    ```
-
-Don't put them into the source folder because then they will be syncroniced back
-into the docker instance into the `/vagrant` directory which might take a
-reasonable amount of resources to do.
-
-The container/virtual machine started via vagrant will sync the cloned git
-repository and use the manifests contained in it to set up the build
-environment. This means that the branch/commit currently checked out will
-determine what version is being built. The final step will copy the image
-directory containing the built images to the directory on the host where vagrant
-was started.
-
-### Using Repo tool
-
-#### NOTE: When using this approach one should not clone this git repository, all git manipulation is handled by the repo tool.
-
-First, install repo tool as instructed at http://source.android.com/source/downloading.html#installing-repo.
-
-Then create a directory for the PELUX build.
-```bash
-mkdir pelux
-cd pelux
-```
-
-Instruct repo tool to fetch a manifest using the command `repo init`. In this
-context, branch denotes what branch of git repo `pelux-manifests` to use. Then
-make repo tool fetch all sources using the command `repo sync`.
-```bash
-repo init -u https://github.com/Pelagicore/pelux-manifests.git -m pelux.xml -b <branch>
-repo sync
-```
-
-When `repo sync` has finished fetching the sources, the next step is to create a
-'build' directory and set up bitbake.  The TEMPLATECONF environment setting
-tells the `oe-init-build-env` script which path to fetch configuration files
-from. Note that the example below gets the template configuration for the Intel
-BSP (without QtAS). Adapt the path according to your current BSP.
-
-```bash
-export TEMPLATECONF=`pwd`/sources/meta-pelux/meta-intel-extras/conf/
-source sources/poky/oe-init-build-env build
-```
-
-The script will create configs if there are no configs present, a message about
-created `conf/local.conf` and `conf/bblayers.conf` files is normal.
-
-Finally, build the desired image. See the variables description above for
-information on the different images.
-```bash
-bitbake <image>
-```
+Full documentation of PELUX is available at [the PELUX Software
+Factory](http://pelux.io/software-factory/) Please refer to the Software Factory
```

Comment:
I think I'd prefere a deeper link to at least http://pelux.io/software-factory/chapters/baseplatform/ or even the whole way to http://pelux.io/software-factory/chapters/baseplatform/building-PELUX-sources.html

Reply:
While that could be useful, I thought not to, since the path might change over time and we would end up with a broken link then.

Reply:
So how does someone understand that they need to click on "Baseplatform" to find anything about bitbake?

Reply:
The frontpage contains a table of contents that has the heading "Building the PELUX sources" in it. I think that suffices.